tools/install_demo_configuration.sh

```diff
@@ -373,7 +373,7 @@ echo "opendistro_security.enable_snapshot_restore_privilege: true" | $SUDO_CMD t
 echo "opendistro_security.check_snapshot_restore_write_privileges: true" | $SUDO_CMD tee -a "$ES_CONF_FILE" > /dev/null
 echo 'opendistro_security.restapi.roles_enabled: ["all_access", "security_rest_api_access"]' | $SUDO_CMD tee -a "$ES_CONF_FILE" > /dev/null
 echo 'opendistro_security.system_indices.enabled: true' | $SUDO_CMD tee -a "$ES_CONF_FILE" > /dev/null
-echo 'opendistro_security.system_indices.indices: [".opendistro-alerting-config", ".opendistro-alerting-alert*"]' | $SUDO_CMD tee -a "$ES_CONF_FILE" > /dev/null
+echo 'opendistro_security.system_indices.indices: [".opendistro-alerting-config", ".opendistro-alerting-alert*", ".opendistro-anomaly-results", ".opendistro-anomaly-detector*"]' | $SUDO_CMD tee -a "$ES_CONF_FILE" > /dev/null
```

Comment:
`.opendistro-anomaly-results` -> `.opendistro-anomaly-results*` ?
`.opendistro-anomaly-detector*` -> `.opendistro-anomaly-detector` ?

Reply:
Sure for results its makes sense as we have other history indices. 
for `.opendistro-anomaly-detector*` it includes both  `.opendistro-anomaly-detector-jobs` and `.opendistro-anomaly-detectors`

Reply:
Got it, make sense.